infra/examples-dev/aws-all/main.tf

```diff
@@ -0,0 +1,161 @@
+terraform {
```

Comment:
what about splitting L1-L26 into `providers.tf` and `state.tf`, that way `main.tf` could be reused too

Reply:
🤔 yes, that's worth thinking about